scep/options.go

```diff
@@ -26,6 +26,9 @@ type Options struct {
 	// are used to be able to load the provisioners when the SCEP authority is being
 	// validated.
 	SCEPProvisionerNames []string
+	// SkipValidation is used to skip the validation of the options, when implementing custom
+	// integrations
+	SkipValidation       bool
```

Comment:
This can remain `SkipValidation`, but update the comment
```suggestion
	// SkipValidation is used to skip the validation of the SCEP options.
	SkipValidation       bool
```